src/Keboola/GoogleDriveWriter/Application.php

```diff
@@ -133,4 +131,31 @@ private function validateParameters(array $parameters): array
             throw new UserException($e->getMessage(), 400, $e);
         }
     }
+
+    private function processTables(): bool
+    {
+        /** @var array<int, mixed> $tables */
+        $tables = $this->container['parameters']['tables'];
```

Comment:
tohle nemusí být vždy nastavený, proto tam bylo původně !empty()

Reply:
tohle není resolvnutý :) problém bude v tom že ti to tady vyhučí na undefined index tables, takže bych tu udělal něco jako

```
$tables = $this->container['parameters']['tables'] ?? [];
```

a pak by tam nemusel být ten if na empty